spec_mysql/models.moon

```diff
@@ -16,20 +16,52 @@ class Users extends Model
 class Posts extends Model
   @timestamp: true
 
+  @relations: {
+    {"images", has_many: "Images"}
+  }
+
   @create_table: =>
     drop_tables @
     create_table @table_name!, {
       {"id", types.id}
       {"user_id", types.integer null: true}
-      {"title", types.text null: false}
-      {"body", types.text null: false}
```

Comment:
Mysql tests were broken without this. It probably slipped since the mysql tests weren't included in ci.sh.